components/Contributors/index.jsx

```diff
@@ -31,7 +31,7 @@ function Contributors({ contributors }) {
   }, [filteredContributors]);
 
   const handleSearch = (e) => {
-    const term = e.target.value.trim().toLowerCase();
```

Comment:
Removing toLowerCase might break the search functionality as we are also using toLowerCase() in the text in which the term has to be searched (see [this](https://github.com/elixir-cloud-aai/landing-page/blob/dev/components/Contributors/index.jsx#L44) to get better understanding). It would be better if we have a separate variable to set in the search bar and keep the current variable as it is.